sqlglot/expressions.py

```diff
@@ -764,7 +764,7 @@ class GeneratedAsIdentityColumnConstraint(ColumnConstraintKind):
 
 
 class NotNullColumnConstraint(ColumnConstraintKind):
-    pass
+    arg_types = {"allow_null": False}
```

Comment:
I didn't realise that postgres.py mentioned NotNullColumnConstraint when I made this change. But I've now reverted this change because I think it makes more sense the way it was originally- having NOT NULL as a constraint is the norm vs having NULL.